.github/workflows/codesee-arch-diagram.yml

```diff
@@ -0,0 +1,23 @@
+# This workflow was added by CodeSee. Learn more at https://codesee.io/
```

Comment:
I'm not sure if this was accidentally added, but can we remove this?

Reply:
Yes, I did that by mistake. I'm removing it